src/containers/treePage/index.tsx

```diff
@@ -126,6 +134,9 @@ const TreePage: React.FC<TreeProps> = ({
     nsMode: 'fallback',
   });
 
+  const [editSiteForm] = Form.useForm();
+  const [mapSearchMarker, setMapSearchMarker] = useState<google.maps.Marker>();
+
```

Comment:
⛏️ I think we can rid of these variables on the `treePage` - they're for keeping the pin in sync with the lat/long fields of the edit site form (pic) on the edit site page. Since we're not editing the site here, we can keep things simple and not worry about keeping them in sync.

![image](https://github.com/Code-4-Community/speak-for-the-trees-frontend/assets/68914997/33c510a8-c837-4cb0-b23f-2a9774fee5e9)